### PR TITLE
Created using Colab

### PR DESCRIPTION
**Pull request by alicechang0909**
<br>
Add FeatureStore Feature Monitoring Notebook
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [X] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [X] Follow the style and grammar rules outlined in the above notebook template.
- [X] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [X] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [X] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [X] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

**Review by kazunori279 — APPROVED** [review]
LGTM!

File: notebooks/official/feature_store/feature_monitoring_with_feature_registry.ipynb [hunk 1]
```diff
@@ -0,0 +1,975 @@
+{
```
[the remark this answers is not in the document]
Comment (alicechang0909) [reply notebooks/official/feature_store/feature_monitoring_with_feature_registry.ipynb:1]: Done

[the remark this answers is not in the document]
Comment (alicechang0909) [reply notebooks/official/feature_store/feature_monitoring_with_feature_registry.ipynb:1]: Done

[the remark this answers is not in the document]
Comment (alicechang0909) [reply notebooks/official/feature_store/feature_monitoring_with_feature_registry.ipynb:1]: Done

[the remark this answers is not in the document]
Comment (alicechang0909) [reply notebooks/official/feature_store/feature_monitoring_with_feature_registry.ipynb:1]: Done

[the remark this answers is not in the document]
Comment (alicechang0909) [reply notebooks/official/feature_store/feature_monitoring_with_feature_registry.ipynb:1]: done

[the remark this answers is not in the document]
Comment (alicechang0909) [reply notebooks/official/feature_store/feature_monitoring_with_feature_registry.ipynb:1]: done

**Review by inardini — APPROVED** [review]
LGTM

**Pull request merged**